src/site/content/en/learn/css/the-cascade/index.md

```diff
@@ -0,0 +1,278 @@
+`a.my-class.another-class[href]:hover` you get something quite hard to overwrite with other CSS.
+For this reason, and to help make your CSS more reusable,
+it's a good idea to keep your selectors as simple as possible.
+Use specificity as a tool to get at elements when you need to,
+but always consider refactoring long, specific selector lists, if you can.
+
+## Origin
+
+The CSS that you write isn't the only CSS applied to a page.
+The cascade takes into account the origin of the CSS.
+This origin includes the browser's internal stylesheet,
+styles added by browser extensions or the operating system,
+and your authored CSS.
+The **order of specificity of these origins**, from least specific, to most specific is as follows:
+
+1. **User agent base styles**. These are the styles that your browser applies to HTML elements by default.
+2. **Local user styles**. These can come from the operating system level, such as a base font size,
+or a preference of reduced motion.
+They can also come from browser extensions,
+such as a browser extension that allows a user to write their own custom CSS for a webpage.
+3. **Authored CSS**. The CSS that you author.
+4. **Authored `!important`**. Any `!important` that you add to your authored declarations.
+5. **Local user styles `!important`**. Any `!important` that come from the operating system level,
+or browser extension level CSS.
+6. **User agent `!important`**. Any `!important` that are defined in the default CSS,
+provided by the browser.
+
+<figure class="w-figure">
+{% Img src="image/VbAJIREinuYvovrBzzvEyZOpw5w1/rjZMVAC5lNATzCErbyeI.png",
+alt="A visual demonstration of the order of origins as also explained in the list.", width="800", height="347" %}
+</figure>
+
+If you have an `!important` rule type in the CSS you have authored
+and the user has an `!important` rule type in their custom CSS, who's CSS wins?
+
+@@TO DO Self assessment
+
+## Importance
+
+Not all CSS rules are calculated the same as each other,
+or given the same specificity as each other.
+
+The **order of importance**, from least important,
+to most important is as follows:
+
+1. normal rule type, such as `font-size`, `background` or `color`
+2. `animation` rule type
+3. `!important` rule type (following the same order as origin)
+4. `transition` rule type
+
+Active animation and transition rule types have higher importance than normal rules.
+In the case of transitions higher importance than `!important` rule types.
+This is because when an animation or transition becomes active,
+its expected behaviour is to change visual state.
+
+## Using DevTools to find out why some CSS is not applying
+
+Browser DevTools will typically show all CSS that could match an element,
+with those which are not being used crossed out.
+
+<figure class="w-figure">
+{% Img src="image/VbAJIREinuYvovrBzzvEyZOpw5w1/LA0lZUYkptGrwrDsLEe1.png",
+alt="An image of browser DevTools with overwritten CSS crossed out", width="800", height="446" %}
```

Comment:
Love the idea behind this image. I'd love to see the text be bigger. There's a lot of whitespace and it's hard to read the DevTools bit.

Reply:
Yeh I'll do a tighter crop on that one